tests/cppunit/t_set_test.cc

```diff
@@ -188,6 +188,27 @@ TEST_F(RedisSetTest, Inter) {
   set->Del(k3);
 }
 
+TEST_F(RedisSetTest, InterCard) {
+  int ret;
```

Comment:
Would you like to also write some unit tests in golang? e.g. `tests/gocase/unit/type/...`